packages/web-react/src/components/Dropdown/DropdownTrigger.tsx

```diff
@@ -1,21 +1,22 @@
 import classNames from 'classnames';
-import React, { ElementType } from 'react';
+import React, { ElementType, ReactNode } from 'react';
 import { useStyleProps } from '../../hooks';
 import { StyleProps } from '../../types';
 import { useDropdownContext } from './DropdownContext';
 import { useDropdownAriaProps } from './useDropdownAriaProps';
 import { useDropdownStyleProps } from './useDropdownStyleProps';
 
-interface DropdownTriggerProps extends StyleProps {
-  elementType?: ElementType | string;
-  children: string | React.ReactNode | ((props: { isOpen: boolean }) => React.ReactNode);
-}
+type DropdownTriggerProps<E extends ElementType> = {
```

Comment:
It is looking good 👍 Just small suggestion for refactoring: can we move this type to other dropdown types to `types/dropdown.ts`, please?